benchopt/runner.py

```diff
@@ -291,7 +292,9 @@ def run_one_solver(benchmark, objective, solver, meta, max_runs, n_repetitions,
             )
 
             solver_strategy = getattr(
-                solver, 'stop_strategy', solver.stopping_criterion.strategy
+                solver,
+                'solver_strategy',
+                solver.stopping_criterion.strategy
```

Comment:
With the current code, as you return `None` if the strategy is not defined in the `Solver`, you would never end up with `solver.stopping_criterion.strategy` here no? You would get `None` I guess.